### ISSUE TITLE
Wallet history cannot be loaded from electrs

### ISSUE THREAD

**Issue by lupn3** [sparrowwallet/sparrow#729]
For quite some time now, the wallet history cannot be loaded from electrs. No transactions are displayed ("No Transactions"). Electrs (or the target computer) no longer responds. Have completely uninstalled and reinstalled electrs several times. Unfortunately, the problem persists.

Btw:
Raspiblitz 1.8.0 running, full synced

Electrs

is activated
has opened port 50002
is connected to localhost

**Comment by craigraw** [comment]
Can you successfully test the the connection in the server preferences? What is the output?

What is the output in the wallet loading panel (Transactions tab)?

**Comment by lupn3** [comment]
Yes, it is called: "Connected to electrs/0.9.7 on protocol version 1.4
Batched RPC enabled.
Server Banner: Welcome to electrs v0.9.7 - the Electrum Rust Server on your RaspiBlitz"

The output in the wallet loading panel says: 

"[Oct 31 16:23:23] Finding transactions for [../0/0-../0/206]
[Oct 31 16:23:39] Failed to subscribe to paths: [../0/0-../0/206]
[Oct 31 16:30:38] Finding transactions for [../0/0-../0/206]
[Oct 31 16:30:51] Finding transactions for [../1/0-../1/199]
[Oct 31 16:30:55] Finished loading."

After refresh: "[Oct 31 16:30:59] Finished loading."

**Comment by lupn3** [comment]
First of all, thank you very very much for the extraordinarily fast profound feedback and your time.

With your advice I am getting closer to the matter. 

Can you kindly tell me which command I can use to view the Electrs logs?
